spec.html

```diff
@@ -19751,95 +19543,17 @@ <h1>Runtime Semantics: Evaluation</h1>
             1. If IsUnresolvableReference(_val_) is *true*, return *"undefined"*.
           1. Set _val_ to ? GetValue(_val_).
           1. [id="step-typeof-web-compat-insertion-point"] NOTE: This step is replaced in section <emu-xref href="#sec-IsHTMLDDA-internal-slot-typeof"></emu-xref>.
-          1. Return a String according to <emu-xref href="#table-typeof-operator-results"></emu-xref>.
+          1. If _val_ is *undefined*, return *"undefined"*.
+          1. If _val_ is a Boolean, return *"boolean"*.
+          1. If _val_ is a Number, return *"number"*.
+          1. If _val_ is a String, return *"string"*.
+          1. If _val_ is a Symbol, return *"symbol"*.
+          1. If _val_ is a BigInt, return *"bigint"*.
+          1. If _val_ is *null*, return *"object"*.
```

Comment:
Seems like it would be nice for the order of this operation and ToString to match. I don't really care what the order is (though I'd prefer string be next to symbol, number next to bigint, undefined next to null, and object last).